aep/general/0135/aep.md.j2

```diff
@@ -0,0 +1,217 @@
+  that the existence of the resource has reached a steady-state and reading
+  resource state returns a consistent response.
+
+{% tab proto -%}
+
+Delete methods are specified using the following pattern:
+
+```proto
+rpc DeleteBook(DeleteBookRequest) returns (library.protobuf.Empty) {
+  option (library.api.http) = {
+    delete: "/v1/{name=publishers/*/books/*}"
+  };
+  option (library.api.method_signature) = "name";
+}
+```
+
+- The RPC's name **must** begin with the word `Delete`. The remainder of the
+  RPC name **should** be the singular form of the resource's message name.
+- The request message **must** match the RPC name, with a `Request` suffix.
+- The response message **should** be `library.protobuf.Empty`.
+  - If the delete RPC is [long-running](#long-running-delete), the response
+    message **must** be a `library.longrunning.Operation` which resolves to the
+    correct response.
+- The request message field receiving the resource name **should** map to the
+  URI path.
+  - This field **should** be called `name`.
+  - The `name` field **should** be the only variable in the URI path. All
+    remaining parameters **should** map to URI query parameters.
+- There **must not** be a `body` key in the `library.api.http` annotation.
+- There **should** be exactly one `library.api.method_signature` annotation,
+  with a value of `"name"`. If an etag or force field are used, they **may** be
+  included in the signature.
+- If the API is operating on the [Management Plane][], the method should have
+  [strong consistency][]: the completion of a delete method **must** mean
+  that the existence of the resource has reached a steady-state and reading
+  resource state returns a consistent response.
+
+Delete methods implement a common request pattern:
+
+```proto
+message DeleteBookRequest {
+  // The name of the book to delete.
+  // Format: publishers/{publisher}/books/{book}
+  string name = 1 [
+    (library.api.field_behavior) = REQUIRED,
+    (library.api.resource_reference) = {
+      type: "library.example.com/Book"
+    }];
+}
+```
+
+- A `name` field **must** be included. It **should** be called `name`.
+  - The field **should** be [annotated as required][aep-203].
+  - The field **must** identify the [resource type][aep-123] that it
+    references.
+- The comment for the field **should** document the resource pattern.
+- The request message **must not** contain any other required fields, and
+  **should not** contain other optional fields except those described in this
+  or another AEP.
+
+{% endtabs %}
+
+### Soft delete
+
+**Note:** This material was moved into its own document to provide a more
+comprehensive treatment: [AEP-164][aep-164].
+
+### Long-running delete
+
+Some resources take longer to delete than is reasonable for a
+regular API request. In this situation, the API **should** use a long-running
+operation instead. See [AEP-151][aep-151] for full details on long-running operations.
```

Comment:
Nit:  should we just linkify the first mention of "long-running operation" and drop the second sentence?

Reply:
done